data/tokens.json

```diff
@@ -1784,6 +1788,10 @@
       "value": "{gtmds_border-radius-max}",
       "type": "borderRadius"
     },
+    "gtmds_avatar-label_font-size-xxl": {
+      "value": "{Header Size.gtmds_font-size-h2}*2",
```

Comment:
I suggest we either use h1 or set a numeric value, it somehow looks better from an architectural point of view to me because with font sizes we have no rule to multiply or do anything like that for the typographies.

Reply:
I think a numeric value is our thing here since this token is really an exception.

Reply:
I agree

Reply:
I think we can co with 64px or even 72px

Reply:
64 is splendid